demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -673,6 +675,22 @@ def is_valid_fetch(self) -> bool:
 
         return fetch_params_exist
 
+    def is_valid_feed_tag(self) -> bool:
```

Comment:
I think this function is not needed, and it can sit under `all_feed_params_exist`